data/.gitignore

```diff
@@ -0,0 +1,3 @@
+/test.txt
```

Comment:
why do we have the dataset uploaded? it is already handled by dvc right?

Reply:
I m not sure what you mean